### PR TITLE
[module] Add follow me module based on ground GPS

### PR DESCRIPTION
**Pull request by fvantienen**

### CODE REVIEW

File: conf/flight_plans/tudelft/rotorcraft_follow_land.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    </block>
+
+    <block name="follow_module">
+      <stay pre_call="follow_me_set_wp(WP_FOLLOW)" wp="FOLLOW"/>
+    </block>
+    <block name="follow_module_land">
+      <stay pre_call="follow_me_set_wp(WP_FOLLOW)" climb="nav_descend_vspeed" vmode="climb" wp="FOLLOW"/>
+    </block>
+
+    <!--block name="stay_p1">
```
[the remark this answers is not in the document]
Comment (dewagter) [reply conf/flight_plans/tudelft/rotorcraft_follow_land.xml]: like this?

[the remark this answers is not in the document]
Comment (tomvand) [reply conf/flight_plans/tudelft/rotorcraft_follow_land.xml]: Yes

**Comment by gautierhattenberger** [comment]
@fvantienen can you update to the correct pprzlink version and fix the conflicts ?

**Comment by fvantienen** [comment]
I will upgrade this later to a better improved version. So this is still WIP

**Comment by OpenUAS** [comment]
To self so @Chris (or FvT...) can finalize it

**Comment by gautierhattenberger** [comment]
Is this still relevant with #2499 ?

**Comment by fvantienen** [comment]
This one can be closed I make a new PR with the rest.

**Pull request closed without merging**